<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="DESKTOP-7F3VJB9" name="com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup" tests="3" failures="1" timestamp="2019-10-26T00:44:40 PDT" time="18.486" errors="0">
  <testcase name="doUserReg" time="5.808" classname="com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup">
    <failure type="java.lang.AssertionError" message="User Reg Test Failed">
      <![CDATA[java.lang.AssertionError: User Reg Test Failed
at org.testng.Assert.fail(Assert.java:97)
at com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup.doUserReg(TestCase1ManualExtentSetup.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doUserReg -->
  <testcase name="doLogin" time="6.797" classname="com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup"/>
  <testcase name="isSkip" time="5.881" classname="com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup">
    <skipped/>
  </testcase> <!-- isSkip -->
</testsuite> <!-- com.test.test07.listeners.onfailure.screenshot.extentreport2.TestCase1ManualExtentSetup -->
